frontend/components/header.js

```diff
@@ -10,7 +11,7 @@ const Highlights = styled.p`
 export default () => (
   <section>
     <Link to="/">
-      <img alt="Shields.io" src="/static/logo.svg" />
+      <Logo />
```

Comment:
Do we lose the alt text with this, or is there some Gatsby magic (or is it baked into the svg already)?

Reply:
I'm not sure! You could check the staging app. This is getting handled by `babel-plugin-inline-react-svg`, which I imagine has a way to provide alt text.

Reply:
I _think_ it is, but I don't think it matters all that much